### PR TITLE
add dotsClass prop in Carousel

### PR DESCRIPTION
**Pull request by krishangoelmt**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/carousel/index.en-US.md](https://github.com/krishangoelmt/ant-design/blob/dotsClass/Carousel/components/carousel/index.en-US.md)

### CODE REVIEW

File: components/carousel/index.tsx [hunk 1]
```diff
@@ -129,12 +131,13 @@ export default class Carousel extends React.Component<CarouselProps, {}> {
...
     }
-
     let className = getPrefixCls('carousel', props.prefixCls);
     const dotsClass = 'slick-dots';
     const dotPosition = this.getDotPosition();
     props.vertical = dotPosition === 'left' || dotPosition === 'right';
-    props.dotsClass = `${dotsClass} ${dotsClass}-${dotPosition || 'bottom'}`;
+    props.dotsClass = `${dotsClass} ${dotsClass}-${dotPosition || 'bottom'} ${
+      this.props.dotsClass
+    }`;
```
Comment (afc163) [inline components/carousel/index.tsx]: Try `classNames`。

Comment (krishangoelmt) [reply components/carousel/index.tsx]: classNames is not any prop then how it will work?

Comment (afc163) [reply components/carousel/index.tsx]: I mean https://www.npmjs.com/package/classnames

Comment (krishangoelmt) [reply components/carousel/index.tsx]: did u get the issue?

**Comment by afc163** [comment]
3.x-stable won't accept new feature now, please send this PR to master or feature branch.

File: components/carousel/index.tsx (hunk 1 above)
Comment (krishangoelmt) [reply components/carousel/index.tsx]: the master branch has some conflicts.

Comment (krishangoelmt) [reply components/carousel/index.tsx]: https://github.com/ant-design/ant-design/pull/21141

**Pull request closed without merging**